apis/swagger.yml

```diff
@@ -2236,7 +2236,17 @@ definitions:
           MaxLength: 64
           The characters of given id should be in 0123456789abcdef.
           By default, given id is unnecessary.
- 
+      MaskedPaths:
```

Comment:
If both the two fields are only used in CRI, how about adding comments `(only used in CRI)` in the description ? @Starnop @fuweid

Reply:
In fact, it's not only for CRI, container engine should also support it. So do with [moby](https://github.com/moby/moby/blob/master/api/swagger.yaml#L786).

Reply:
OK, got it.